app/2.1.x/hybrid-mode.md

```diff
@@ -27,6 +27,31 @@ Deploying using Hybrid mode has a number of benefits:
 * Easiness of management, since an admin will only need to interact with the CP nodes to control
   and monitor the status of the entire Kong cluster.
 
+## Configuration Properties
+
+Hybrid Mode introduces the following configuration properties:
+- `cluster_listen`: List of addresses and ports on which the Control Plane will listen
+for incoming Data Plane connections. Defaults to `0.0.0.0:8005`.
+- `cluster_control_plane`: Port on which the Data Plane nodes will attempt to connect
+to the Control Plane. Must contain the `cluster_listen` value that was configured in
```

Comment:
```suggestion
to the Control Plane. Must point to the `cluster_listen` value that was configured in
```

The `cluster_control_plane` and `cluster_listen` values are likely not exactly the same, so "point to" may be more appropriate.